packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -140,9 +136,9 @@ contract SystemConfig is OwnableUpgradeable, ReinitializableBase, ISemver {
     event ConfigUpdate(uint256 indexed version, UpdateType indexed updateType, bytes data);
 
     /// @notice Semantic version.
-    /// @custom:semver 2.6.0
+    /// @custom:semver 2.7.0
     function version() public pure virtual returns (string memory) {
-        return "2.6.0";
+        return "2.7.0";
```

Comment:
Per our style guide this should be a major bump because it breaks the interface, I don't know if we're usually consistent about that though https://github.com/ethereum-optimism/optimism/blob/1447c93b2165179b56387a7f8b872dfdf1b57b99/packages/contracts-bedrock/book/src/contributing/style-guide.md#L128

Reply:
I'm fine either way, do you have a strong preference?

Reply:
No strong preference, maybe @maurelian does?

Reply:
Glad we did use, we should stick to our processes